src/configure_settings/configure_settings.jl

```diff
@@ -199,3 +208,132 @@ function configure_settings_multistage(settings_path::String)
+    "esr_name" => "Energy_share_requirement.csv",
+    "esr_slack_name" => "Energy_share_requirement_slack.csv",
+    "min_cap_name" => "Minimum_capacity_requirement.csv",
+    "max_cap_name" => "Maximum_capacity_requirement.csv",
+    "operational_reserves_name" => "Operational_reserves.csv")
+end
+
+@doc raw"""
+    configure_input_names(case::AbstractString)
+
+Reads in the settings from the `input_settings.yml` YAML file and
+merges them with the default input settings. It then returns the
+settings dictionary.
+
+# Arguments
+- `case::AbstractString`: The case containing the settings file.
+
+# Returns
+- `names::Dict`: The input names dictionary.
+"""
+function configure_input_names(case::AbstractString)
+    println("Configuring Input File and Path Names")
+    input_settings_path = get_settings_path(case, "input_settings.yml")
+    input_names = YAML.load(open(input_settings_path))
+    input_names = isfile(input_settings_path) ? YAML.load(open(input_settings_path)) : Dict{Any, Any}()
+
+    names = default_input_names(case)
+    merge!(names,input_names)
+
+    return names
+end
+
+function default_results_names()
+   Dict{Any, Any}("capacity_name" => "capacity",
+    "capacity_factor_name" => "capacityfactor",
+    "captured_emissions_plant_name" => "captured_emissions_plant",
+    "charge_name" => "charge.csv",
+    "charging_cost_name" => "ChargingCost",
+    "co2_prices_name" => "CO2_prices_and_penalties",
+    "commit_name" => "commit",
+    "costs_name" => "costs",
+    "curtail_name" => "curtail",
+    "emissions_plant_name" => "emissions_plant",
+    "emissions_name" => "emissions",
+    "energy_revenue_name" => "EnergyRevenue",
+    "flow_name" => "flow",
+    "fuel_cost_plant_name" => "Fuel_cost_plant",
+    "fuel_consumption_plant_name" => "FuelConsumption_plant_MMBTU",
+    "fuel_consumption_total_name" => "FuelConsumtion_total_MMBTU",
+    "mincap_name" => "MinCapReq_prices_and_penalties",
+    "revenue_name" => "NetRevenue",
+    "network_expansion_name" => "network_expansion",
+    "nse_name" => "nse",
+    "power_balance_name" => "power_balance",
+    "power_name" => "power",
+    "prices_name" => "prices",
+    "reg_subsidy_revenue_name" => "RegSubsidyRevenue",
+    "reg_name" => "reg",
+    "reg_dn_name" => "reg_dn",
+    "reliability_name" => "reliability",
+    "shutdown_name" => "shutdown",
+    "start_name" => "start",
+    "status_name" => "status",
+    "storage_name" => "storage",
+    "subsidy_revenue_name" => "SubsidyRevenue",
+    "time_weights_name" => "time_weights",
+    "tlosses_name" => "tlosses",
+    "virtual_discharge_name" => "virtual_discharge",
+    "vre_stor_dc_charge_name" => "vre_stor_dc_charge",
+    "vre_stor_ac_charge_name" => "vre_stor_ac_charge",
+    "vre_stor_dc_discharge_name" => "vre_stor_dc_discharge",
+    "vre_stor_ac_discharge_name" => "vre_stor_ac_discharge",
+    "vre_stor_wind_power_name" => "vre_stor_wind_power",
+    "vre_stor_solar_power_name" => "vre_stor_solar_power")
+end
```

Comment:
All of these are `_name`s. Since we're working with an `results_names` Dict here that should be clear enough from context. Consider just calling them `"capacity_factor"` and so on. There's nothing wrong with having the same entry as key and value, like `"storage" => "storage"`.

As an alternate suggestion you might want to make this a Dict of `Symbol => AbstractString`.
Strings are preferred when they need to be manipulated in some way, but especially in the keys these are just symbolic signifiers.